README.md

```diff
@@ -27,33 +27,68 @@ See the next section about to get started, the package documentation, the
 
 ### Getting Started
 
+#### Installing drat
+
 You can install the package from [CRAN](http://cran.r-project.org) via the
-standard `install.packages("drat")`.  Alternatively, for a first
-installation, you can also do
+standard `install.packages("drat")`. See below for a way to install drat 
+from it's own drat repository.
 
-```{.r}
-install.packages("drat", repos="http://eddelbuettel.github.io/drat")
-``` 
+#### Adding packages to a new drat repository
 
-to install directly from the drat repository.  After either initial
-installation, you can add line such as
+The following assumes that you want to add a drat repository to your github pages. 
+The github pages are checked out at `/src/repo` and available on the web 
+at <github_username>.github.io.
+
+As a first step, add a `drat` subdirectory (`/src/repo/drat`). This is where 
+new packages should be added to.
+
+Package insertion into a repository then works by passing the package tarball
+filename as an argument to `insertPackage()` or its shorthand variant:
 
 ```{.r}
-drat:::add("eddelbuettel")
+drat:::insert("drat_0.0.1.tar.gz", repodir="/src/repo/drat")
 ```
 
-to your session or startup files (see `help(Startup)`) and use
-`update.packages()` as usual---also reflection the new drat archive.
+Please note, that you can specify a default repository via 
+`options(dratRepo="/src/repo/drat")` and then only pass in the filename of 
+the package.
 
-Package insertion into a repo then works by passing the package tarball
-filename as an argument to `insertPackage()` or its shorthand variant:
+Now you can now `git add`, `commit` and `push` the new version of your github 
+pages.
+
+Please note, that there is currently only support for source packages, not Windows
+Mac OSX binary packages.
+
+#### Using your new drat repository
+
+There are two ways to install packages from this new drat package repository:
+
+Directly by specifying the `repos` argument to `install.packages()`:
+
+```{.r}
+install.packages("<package>", repos="http://<github_username>.github.io/drat", type="source")
```

Comment:
? 
I thought that `install.packages()` only reads from the repo?

It's also that way in the current README: `install.packages("drat", repos="http://eddelbuettel.github.io/drat")`